music21/stream/__init__.py

```diff
@@ -8704,7 +8684,6 @@ def sliceByGreatestDivisor(self, *, addTies=True, inPlace=False):
             # call on component measures
             for m in returnObj.getElementsByClass('Measure'):
                 m.sliceByGreatestDivisor(addTies=addTies, inPlace=True)
-            returnObj.coreElementsChanged()
```

Comment:
oh my...this routine should be moved off of stream to somewhere else in v7.  :-)  No one ever uses this.